index.js

```diff
@@ -1 +1,4 @@
-// code your solution here
+const superbowlWin = (record) => {
+  const winRecord = record.find(item => item.result === 'W')
+  return winRecord === undefined ? undefined : winRecord.year
```

Comment:
Even simpler with the same result:
```suggestion
  return if (winRecord) { return winRecord.year }
```